modules/javafx.graphics/src/test/java/test/javafx/scene/layout/SnappingTest.java

```diff
@@ -119,46 +129,46 @@ private void testContainerSnappingImpl(ContainerInstruction<Region> containerIns
             assertEquals(snappedPaddingX, container.minWidth(-1), 0.00001, className);
             assertEquals(snappedPaddingY, container.minHeight(-1), 0.00001, className);
         } else {
-            assertEquals(widthHeight + snappedPaddingX, container.minWidth(-1), 0.00001, className);
-            assertEquals(widthHeight + snappedPaddingY, container.minHeight(-1), 0.00001, className);
+            assertEquals(widthHeight + snappedPaddingX, container.minWidth(-1), DELTA, className);
+            assertEquals(widthHeight + snappedPaddingY, container.minHeight(-1), DELTA, className);
         }
 
-        assertEquals(widthHeight + snappedPaddingX, container.prefWidth(-1), 0.00001, className);
-        assertEquals(widthHeight + snappedPaddingY, container.prefHeight(-1), 0.00001, className);
+        assertEquals(widthHeight + snappedPaddingX, container.prefWidth(-1), DELTA, className);
+        assertEquals(widthHeight + snappedPaddingY, container.prefHeight(-1), DELTA, className);
```

Comment:
sorry, should have specified in my previous comment - this constant is typically called EPSILON (a Greek letter denoting a very small number)

Reply:
While this is correct, the `assertEquals` method in JUnit calls this variable `delta`, so I think both namings are understandable and fine

Reply:
Mathematically speaking, it is not a delta (change), it is a small quantity (epsilon).  it's a minor nit pick, but in this context it should be an EPSILON.

(There is one in TableUtil, but search in the code base and you'll find ~ 4K matches